pages/managed-cards/_id/edit.vue

```diff
@@ -1,7 +1,16 @@
 <template>
   <section>
     <b-container>
-      <b-row>
+      <b-row v-if="pendingDataOrError">
```

Comment:
Ideally we make a difference between pending state and error state, so user will not think something is still loading when there is an error

Reply:
this should be a refactoring task as there are other places where pendingDataOrError is used.